packages/eas-cli/src/uploads.ts

```diff
@@ -57,26 +57,30 @@ async function obtainS3PresignedPostAsync(
 export async function uploadWithPresignedPostAsync(
   stream: Readable | Buffer,
   presignedPost: PresignedPost,
-  handleProgressEvent: ProgressHandler
-) {
+  handleProgressEvent?: ProgressHandler
```

Comment:
Allow handleProgressEvent to be optional. Uploading 100's of assets would look chaotic.